Beginner_concepts/enums.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.6.7;
+
+contract enumTut {
+        //enums
```

Comment:
indentation of comment, maybe use a few blank lines for readablility

Reply:
okay,i'll take care of that futurere